app/scripts/platforms/extension.js

```diff
@@ -223,7 +223,7 @@ export default class ExtensionPlatform {
   }
 
   _viewOnEtherscan (txId) {
-    if (txId.startsWith('http://')) {
+    if (txId.startsWith('http://') || txId.startsWith('https://')) {
```

Comment:
We could probably check only for HTTPS, we shouldn't be using non-TLS links anywhere:

```suggestion
    if (txId.startsWith('https://')) {
```